targets/gnome

```diff
@@ -9,11 +9,32 @@ CHROOTBIN='crouton-noroot startgnome gnome-session-wrapper'
 . "${TARGETSDIR:="$PWD"}/common"
 
 ### Append to prepare.sh:
-install --minimal evolution-data-server gnome-control-center \
-        gnome-screensaver gnome-session gnome-session-fallback \
-        gnome-shell gnome-themes-standard gvfs-backends nautilus \
-        unzip gtk2-engines-pixbuf
+check_distro() 
+{
+    ###list of distros that need gnome-session-fallback
+    set "squeeze" "wheezy" "kali" "trusty" "precise" 
+    name=$(lsb_release -cs)
+    #echo "$name"
+    for i do
+	if [ "$i" = "$name" ]; then
+	   echo "1" #The distro installed needs gnome-session-fallback
+	fi
+    done
+}
+fallback_need=$(check_distro)
 
```

Comment:
You probably didn't know this, but there is a global release check function https://github.com/dnschneid/crouton/blob/master/installer/functions#L123
So you can just use something like
 ```
if release -ge wheezy -ge kali -ge trusty; then
    package-gnome-session="gnome-session-flashback"
else
   package-gnome-session="gnome-session-fallback"
fi
install --minimal evolution-data-server gnome-control-center \
          gnome-screensaver gnome-session $package-gnome-session \
          gnome-shell gnome-themes-standard gvfs-backends nautilus \
          unzip gtk2-engines-pixbuf
``` 
As you see I used a variable so you don't need the repetitive code below.
Thanks for contributing.